ui/lib/core/addon/components/form-field.hbs

```diff
@@ -187,17 +187,38 @@
       @value={{or (get @model this.valuePath) @attr.options.defaultValue}}
       @onChange={{this.onChangeWithEvent}}
     />
+  {{else if (eq @attr.options.editType "toggleButton")}}
+    {{! Togglable Input }}
+    <Toggle
+      @name="toggle-{{@attr.name}}"
+      @onChange={{this.toggleButton}}
+      @checked={{this.toggleInput}}
+      data-test-toggle={{@attr.name}}
+    >
+      <span class="has-text-weight-bold is-large">{{this.labelString}}</span><br />
+      <div class="description has-text-grey">
+        {{#if this.toggleInput}}
+          <span>
```

Comment:
nit: this `<span>` could go outside the `{{#if}}` statement and we wouldn't need to repeat it in both conditionals